aql/aql.proto

```diff
@@ -14,7 +14,7 @@ message GraphStatement {
     oneof statement {
         string V = 1;
         string E = 2;
-        string label = 3;
+        google.protobuf.ListValue labeled = 3;
```

Comment:
dumb question:  why use ` google.protobuf.ListValue` here instead of the more standard `repeated` ?

Reply:
Can't have repeated in a 'oneof'. We've asked, they are keeping it that way. This doesn't typecheck the elements of the array, but it 'looks correct' in JSON.

Reply:
Thanks, learned something new.